app/components/badge-maintenance.js

```diff
@@ -0,0 +1,46 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { alias } from '@ember/object/computed';
+
+export default Component.extend({
+    tagName: 'span',
+    classNames: ['badge'],
+    escapedStatus: computed('badge', function() {
+        return this.get('badge.attributes.status').replace('-', '--');
```

Comment:
So I tried out all of the values for the badges, and `looking-for-maintainer` didn't load. It looks like it's because this `replace` only handles the *first* hyphen:

<img width="1280" alt="screen shot 2017-08-26 at 1 05 53 pm" src="https://user-images.githubusercontent.com/193874/29743417-bcb9796a-8a5f-11e7-82b4-35cc950629ba.png">

Take a look at [the replace docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace) and look for "global" for how to fix this :)